organization-bootstrap/business-units/modules/business-unit-folders/README.md

```diff
@@ -0,0 +1,35 @@
+# Two-levels folders tree
+
+This module creates one top-level folder and set of second-level folders, which meant to represent a business unit with set of enviroments underneeth. It also creates per environment GCS bucket and ServiceAccount in a project specified by `tf_project_id` variable. 
+
+This set of Terraform files is usually applied manually by an org-level administrator as a first step, and then reapplied only when a new environment needs to be created or an existing one removed, and serves different purposes:
+
+## Managed resources and services
+
+This sample creates several distinct groups of resources:
+
+- one top level folder
+- set of second level folders
+- set of GCS buckets, which meant to be used for storing terraform state. 
```

Comment:
The README still seems to describe the previous version with GCS buckets etc. Let me fix that.

Reply:
Thank you